fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -794,6 +796,14 @@ public static boolean analyzeUniqueKeyMergeOnWrite(Map<String, String> propertie
      */
     public static void checkCatalogProperties(Map<String, String> properties, boolean isAlter)
             throws AnalysisException {
+        // firstly, check if the property's key contains empty character
+        for (Map.Entry<String, String> entry : properties.entrySet()) {
```

Comment:
Not a good solution.
It cannot handle all situation, like containing other charactors.